content/docs/reference/community-packages.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Community Packages
+meta_desc: A list of Pulumi packages maintained by the Community
+menu:
+  reference:
+    weight: 6
+---
+
+Name | Cloud | Languages| Description
```

Comment:
I think it might be easier to actually leave off the `Provider` column for now? The packages are usually self-describing in terms of provider (or we can make it clear in the description) and there are provider-agnostic packages like https://github.com/neoskop/pulumi-utils as well as multi-provider repos like https://github.com/materya/pulumi